### ISSUE TITLE
Fresh install does not compile

### ISSUE THREAD

**Issue by SirGordon** [lapanti/ts-react-boilerplate#97]
Just try to clone and build it.
I get a lot of errors like:
```
TypeError: compiler.createWatchProgram is not a function
TypeError: Cannot read property 'updateRootFileNames' of undefined
TypeError: Cannot read property 'getProgram' of undefined
```
etc..
Didn't dig deep but i guess its because dependencies..

**Comment by lapanti** [comment]
yeah sounds like it, sorry I haven't had the time lately to take a look at this but will try during the weekend/early next week to update the whole thing

**Comment by SirGordon** [comment]
That would be awsome!
In the meantime, could you please provide dependency versions that works?

**Comment by lapanti** [comment]
https://github.com/lapanti/ts-react-boilerplate/tree/feature/hn-reader <- this is an in-progress branch I started (to totally revise the boilerplate's functionality), but the dependencies work now (there are a a few modification as well)
